src/libraries/System.Text.Json/tests/Serialization/CyclicTests.cs

```diff
@@ -57,7 +58,8 @@ public static void WriteCyclicFail(int objectHierarchyDepth, int maxDepth, int e
             // Since the last Parent property is null, the serializer moves onto the Children property.
             string expectedPath = "$" + string.Concat(Enumerable.Repeat(".Parent", expectedPathDepth));
             Assert.Contains(expectedPath, ex.Path);
-            Assert.Contains(maxDepth.ToString(), ex.ToString());
+            string expectedMessage = $"the object depth is larger than the maximum allowed depth of {effectiveMaxDepth}";
+            Assert.Contains(expectedMessage, ex.Message);
```

Comment:
```suggestion
            Assert.Contains(effectiveMaxDepth.ToString(), ex.Message);
```

We only assert against targeted parts of the exception messages i.e the content wrapped in parentheses in the `SR` formatted string. For example, see the [string for the exception thrown here](https://github.com/dotnet/runtime/blob/5e6b441047c43b39d0306b21c0b78e0d843742a9/src/libraries/System.Text.Json/src/Resources/Strings.resx). This is to align with potential future globalization efforts where the rest of the string will be translated to various languages.

Reply:
Ah, I see. I'll change it back.

Reply:
Done